cmake/modules/OmrDetectSystemInformation.cmake

```diff
@@ -198,19 +198,28 @@ macro(omr_detect_system_information)
 				# just use GNU config
 				set(_OMR_TOOLCONFIG "gnu")
 			endif()
+		elseif(CMAKE_C_COMPILER_ID MATCHES "IBMClang$")
+			set(_OMR_TOOLCONFIG "openxl")
+			set(CMAKE_C_COMPILER_IS_OPENXL ON CACHE BOOL "ibm-clang is the C compiler")
 		elseif(CMAKE_C_COMPILER_ID MATCHES "^XL(Clang)?$" OR CMAKE_C_COMPILER_ID STREQUAL "zOS")
 			# In CMake 3.14 and prior, XLClang uses CMAKE_C_COMPILER_ID "XL"
 			# In CMake 3.15 and beyond, XLClang uses CMAKE_C_COMPILER_ID "XLClang"
 			set(_OMR_TOOLCONFIG "xlc")
```

Comment:
This belongs within the `if` block for xlc.

Reply:
Just to confirm, does this mean the check would be updated as such:
```
elseif(CMAKE_C_COMPILER_ID MATCHES "^XL(Clang)?$" OR  CMAKE_C_COMPILER_ID MATCHES "IBMClang$" OR CMAKE_C_COMPILER_ID STREQUAL "zOS") 
```

I only had it as a separate clause because it could not enter the `elseif` mentioned above, and the COMPILER_ID was ending up to be `IBMClang` when running with Open XL via a jenkins build.

Reply:
Have updated rest of feedback @keithc-ca , just awaiting a bit of clarification for the above change.